File: Ejercicios Laragon/proyectonodejs2644590/Hypotenuse/hypotenuse.js
function calcular(){

    let lado1 = document.getElementById("lado1").value
    let lado2 = document.getElementById("lado2").value

    let hipotenusa = Math.sqrt(lado1**2 + lado2**2);

    document.getElementById("mensaje").innerHTML = `Hypotenuse is ${hipotenusa}`
}


        

File: Ejercicios Laragon/proyectopython2644590/hypotenuse.py
from math import sqrt

lado1 = float(input("Input the value of the first hick: "))
lado2 = float(input("Input the value of the second hick: "))

hipotenusa = sqrt(lado1**2+lado2**2)

print(f"The hypotenuse is {hipotenusa}")